.github/workflows/reusable-docker-arm-build.yaml

```diff
@@ -59,17 +59,16 @@ jobs:
           repository: quay.io/kairos/packages-arm64
           packages: system/kairos-agent
       - name: Set Version
+        if: ${{ startsWith(github.ref, 'refs/tags/v') || github.event_name == 'pull_request' }}
         run: |
           echo "GIT_VERSION=$(git describe --always --tags --dirty)" >> $GITHUB_ENV
-      - name: Set Image name (master)
+      - name: Set Version
         if: ${{ github.ref == 'refs/heads/master' }}
         run: |
-          IMAGE_REF=$(SOFTWARE_VERSION=${{ inputs.k3s_version }} SOFTWARE_VERSION_PREFIX=k3s FLAVOR=${{ inputs.flavor }} FLAVOR_RELEASE="${{ inputs.flavor_release }}" MODEL=${{ inputs.model }} TARGETARCH=arm64 VARIANT=${{ inputs.variant }} REGISTRY_AND_ORG="${{ env.REGISTRY }}" RELEASE=master kairos-agent versioneer container-artifact-name)
-          echo "IMAGE_REF=${IMAGE_REF}" >> $GITHUB_ENV
-      - name: Set Image name (release/PR)
-        if: ${{ startsWith(github.ref, 'refs/tags/v') || github.event_name == 'pull_request' }}
+          echo "RELEASE=master" >> $GITHUB_ENV
+      - name: Set Image name
         run: |
-          IMAGE_REF=$(SOFTWARE_VERSION=${{ inputs.k3s_version }} SOFTWARE_VERSION_PREFIX=k3s FLAVOR=${{ inputs.flavor }} FLAVOR_RELEASE="${{ inputs.flavor_release }}" MODEL=${{ inputs.model }} TARGETARCH=arm64 VARIANT=${{ inputs.variant }} REGISTRY_AND_ORG="${{ env.REGISTRY }}" RELEASE=${{ env.GIT_VERSION }} kairos-agent versioneer container-artifact-name)
+          IMAGE_REF=$(SOFTWARE_VERSION=v${{ inputs.k3s_version }} SOFTWARE_VERSION_PREFIX=k3s FLAVOR=${{ inputs.flavor }} FLAVOR_RELEASE="${{ inputs.flavor_release }}" MODEL=${{ inputs.model }} TARGETARCH=arm64 VARIANT=${{ inputs.variant }} REGISTRY_AND_ORG="${{ env.REGISTRY }}" RELEASE=${{ env.RELEASE }} kairos-agent versioneer container-artifact-name)
```

Comment:
nope...It has to do this only when there is k3s_version set (otherwise it sets the SOFTWARE_VERSION for core images to just `v`)